internal/rpmostreeclient/rpmostreeclient.go

```diff
@@ -0,0 +1,91 @@
+package rpmostreeclient
```

Comment:
I would name the package `ostree`. We might add other ostree related commands here.

Reply:
updated